be/src/vec/functions/like.cpp

```diff
@@ -499,9 +496,6 @@ Status FunctionLikeBase::hs_prepare(FunctionContext* context, const char* expres
         hs_free_database(*database);
         *database = nullptr;
         *scratch = nullptr;
-        if (context) {
```

Comment:
please leave a comment here, avoid somebody add these codes back again next time.

Reply:
OK，I will add some test and comment.